### PR TITLE
Fix incorrect time measurement in closeLightyModule function

### PR DESCRIPTION
**Pull request by Tobianas**
The previous implementation of `closeLightyModule` method used a Stopwatch to measure the time taken by `lightyModule.shutdown()` method, but it did not account for the time taken by the method to complete asynchronously.

JIRA:LIGHTY-169

### CODE REVIEW

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java [hunk 1]
```diff
@@ -111,13 +115,22 @@ void shutdown() {
...
         if (lightyModule != null) {
             final Stopwatch stopwatch = Stopwatch.createStarted();
+            LOG.info("Lighty module {} shutting down ...", lightyModule);
             try {
-                LOG.info("Lighty module {} shutting down ...", lightyModule);
-                lightyModule.shutdown();
+                final ListenableFuture<Boolean> future = lightyModule.shutdown();
+                Futures.addCallback(future, new FutureCallback<>() {
+                    @Override
+                    public void onSuccess(Boolean result) {
```
Comment (ihrasko) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java:123]: We can get `true` (closed successfully) but also `false` (closed unsuccessfully).

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java (hunk 1 above)
Comment (Tobianas) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java:123]: Added return value.

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java [hunk 2]
```diff
@@ -111,13 +115,26 @@ void shutdown() {
...
+                        if (result) {
+                            LOG.info("Lighty module {} stopped in {}", lightyModule, stopwatch.stop());
+                        } else {
+                            LOG.info("Lighty module {} failed to stop", lightyModule);
```
Comment (ihrasko) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java:127]: `LOG.info("Lighty module {} failed to stop after {}", lightyModule, stopwatch.stop());`

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java [hunk 3]
```diff
@@ -111,13 +115,26 @@ void shutdown() {
     void closeLightyModule(final LightyModule lightyModule) {
```
Comment (ihrasko) [inline lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java:115]: `private static`

Comment (ihrasko) [reply lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java:115]: Also change line 105 to:
`Runtime.getRuntime().addShutdownHook(new Thread(this::shutdown));`

**Review by ihrasko — CHANGES_REQUESTED** [review]

**Review by ihrasko — APPROVED** [review]

**Pull request merged**